docs/concepts/jcloud/yaml-spec.md

```diff
@@ -267,25 +267,25 @@ emphasize-lines: 5-6
 ---
 jtype: Flow
 executors:
-  - uses: jinahub+docker://Sentencizer
+  - uses: jinaai+docker://Sentencizer
     replicas: 5
     jcloud:
       minAvailable: 2
 ```
-> In case of unlike disruption, ensure at least two replicas will still be available, while 3 may be down.
+> In case of disruption, ensure at least two replicas will still be available, while three may be down.
 
 ```{code-block} yaml
 ---
 emphasize-lines: 5-6
 ---
 jtype: Flow
 executors:
-  - uses: jinahub+docker://Sentencizer
+  - uses: jinaai+docker://Sentencizer
```

Comment:
Is this missing the username portion?
![image](https://user-images.githubusercontent.com/4182659/215782787-45d56346-e115-49d9-bcd3-e5c9e6b12419.png)

https://cloud.jina.ai/executor/c6focg47